### PR TITLE
Fix/refactor structure

### PR DESCRIPTION
**Pull request by tetogomez**
### What does this PR do?

- Update dependencies and Refactor Structure

### Steps to test

1. run `make start`
2. go to localhost:3000

#### CheckList

- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: webapp/src/components/Sidebar/styles.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { important } from 'polished'
```
Comment (xavier506) [inline webapp/src/components/Sidebar/styles.js:1]: @tetogomez  see codefactor comment

File: webapp/package.json [hunk 2]
```diff
@@ -27,14 +27,14 @@
     "@apollo/client": "~3.4.0",
```
Comment (xavier506) [inline webapp/package.json:27]: I think we should set this to `"@apollo/client": "^3.4.0",`

**Review by xavier506 — APPROVED** [review]
Awesome Work! 😎

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Pull request merged**